app/public/cantusdata/templates/missing_manuscript_redirect.html

```diff
@@ -0,0 +1,25 @@
+{% extends "base.html" %}
+
+{% block body %}
+
+<div class="container">
+    <div class="row" style="padding-top:20vh;padding-bottom:20vh">
+        <div class="col-lg-12">
+            <p class="text-center">This manuscript has been moved.</p>
+            <p class="text-center">You can find it on <a href="{% url 'manuscript-list' %}">the Manuscripts page</a>.</p>
```

Comment:
should the link be applied to "Manuscripts page", or even just "Manuscripts"? Including the article feels weird to me.

Both the title and the path of the page in question use just "manuscripts". Suggesting you take the "the" and the "page" out of the link.